File: Olympiade/avg2.c
// NOTE: it is recommended to use this even if you don't understand the following code.

#include <assert.h>
#include <stdio.h>
#include <stdlib.h>

#define MAXN 200000

int N, i;
long long C, K;
long long P[MAXN];

int main()
{
    // uncomment the two following lines if you want to read/write from files
    freopen("input1.txt", "r", stdin);
    // freopen("output.txt", "w", stdout);

    assert(2 == scanf("%d%lld", &N, &K));

    for (i = 0; i < N; ++i)
        assert(1 == scanf("%lld", &P[i]));

    C = 0;

    for (int i = 0; i < N; i++)
    {
        printf("%lld ", P[i]);
    }

    printf("%lld\n", C);

    return 0;
}

File: Info/tffhgdhgd/src/avg2.java
// NOTE: it is recommended to use this even if you don't understand the following code.

import java.util.*;
import java.io.*;
import java.lang.*;

public class avg2 {

    static double calcavarege(long[] P) {
        double avarege = 0;
        for (long l : P) {
            avarege += l;
        }
        return avarege / P.length;
    }

    public static void main(String[] args) throws FileNotFoundException, IOException {
        Locale.setDefault(Locale.US);
        InputStream fin = System.in;
        OutputStream fout = System.out;
        // uncomment the two following lines if you want to read/write from files
        // fin = new FileInputStream("input.txt");
        // fout = new FileOutputStream("output.txt");

        Scanner scn = new Scanner(fin);
        PrintStream prnt = new PrintStream(fout);

        int N = scn.nextInt();
        long K = scn.nextLong();

        long[] P = new long[N];
        for (int i = 0; i < N; ++i)
            P[i] = scn.nextLong();

        long C = 0;

        while (calcavarege(P) == K) {
            for (int i = 0; i < P.length; i++) {
                P[i] = -1;
            }

            prnt.format("%d\n", C);

            fout.flush();
        }
    }
}